src/components/input/multimodal/uploader/uploader.tsx

```diff
@@ -151,22 +153,39 @@ function Uploader(props: UploaderProps) {
     setAddedFiles((prev) => prev.filter((file) => file.id !== id))
   }
 
+  function updateDuplicateFileName(file: File) {
```

Comment:
Would the following be a more accurate name?
```suggestion
  function updateFileNameIfDuplicate(file: File) {
```